shell/platform/darwin/ios/framework/Source/FlutterEngine.mm

```diff
@@ -203,6 +203,12 @@ - (void)setFlutterViewControllerWillDeallocObserver:(id<NSObject>)observer {
 - (void)notifyViewControllerDeallocated {
   if (!_allowHeadlessExecution) {
     [self destroyContext];
+  } else {
+    PlatformViewIOS platform_view = [self iosPlatformView];
+    if (platform_view) {
+      fml::WeakPtr<FlutterViewController> null_view_controller;
+      platform_view->SetOwnerViewController(null_view_controller);
```

Comment:
Can this just be `platform_view->SetOwnerViewController({});`?

Reply:
done